docs: remove references to serf.io site (#24114)
hashicorp/nomad: website/content/docs/concepts/gossip.mdx

# Gossip Protocol

> Nomad uses a [gossip protocol](https://en.wikipedia.org/wiki/Gossip_protocol) to
> manage membership. This is provided through the use of the [Serf library][serf].

Reviewer: ```suggestion
Nomad uses the [Serf library][serf] to provide a [gossip protocol](https://en.wikipedia.org/wiki/Gossip_protocol) to
manage membership.
```
active voice

## Gossip in Nomad

Nomad makes use of a single global WAN gossip pool that all servers participate
in.  Membership information provided by the gossip pool allows servers to
perform cross region requests. The integrated failure detection allows Nomad to
gracefully handle an entire region losing connectivity, or just a single server

> in a remote region. The gossip protocol is also used to detect servers in the

Reviewer: ```suggestion
in a remote region. Nomad also uses the gossip protocol to detect servers in the
```
active voice

same region to perform automatic clustering via the [consensus
protocol](/nomad/docs/concepts/consensus).

> All of these features are provided by leveraging [Serf][serf]. It is used as an
> embedded library to provide these features. From a user perspective, this is not
> important, since the abstraction should be masked by Nomad. It can be useful
> however as a developer to understand how this library is leveraged.

Reviewer: ```suggestion
To provide all these features, Nomad uses [Serf][serf] as an
embedded library. From a user perspective, this is not
important, since Nomad masks the abstraction. However, developers may find it useful
to understand how Nomad leverages the Serf library.
```
